### DISCUSSION TITLE
Commander Core

### DISCUSSION THREAD

**Discussion by DaJRJesus** [Rem0o/FanControl.Releases#1329] [Q&A]
Hi, Im trying to get Fan control to work with my Commander Core I have the commander plugin and my core fans rpms show up but I cannot control them even pairing the speed sensor manually to the system fans plz help!
[Screenshot_76](https://user-images.githubusercontent.com/92175145/204630851-2c4094f9-287c-4d3d-82fc-4a461e31ba05.png)

**Comment by Rem0o** [comment]
Do you have ICUE running at the same time? It's one or the other.

**Reply by DaJRJesus** [reply]
I do not I have no other fan software currently running

**Reply by DaJRJesus** [reply]
> I do not I have no other fan software currently running

if you need any other information such as specs etc to solve this issue, please let me know

**Reply by Rem0o** [reply]
You are using the plugin right? Sadly, the plugin is no longer maintained, it was deleted from Github actually, so I can't offer support for it really.

**Reply by DaJRJesus** [reply]
I am it ls probably an issue with the plug in I might just have to bite the bullet get rid of the controller completely and just kinda daisy chain all my fans to the motherboard, I appreciate you trying to help though!

**Reply by DaJRJesus** [reply]
well actually come to find out I have 6 fan headers on my motherboard, so I'll just plug my fans into them and leave the rgb connecters on the hub!
